roxctl/deployment/check/check.go

```diff
@@ -240,7 +240,7 @@ func (d *deploymentCheckCommand) checkDeployment() error {
 
 	alerts, ignoredObjRefs, remarks, err := d.getAlertsAndIgnoredObjectRefs(combinedDeployments)
```

Comment:
This command also has the issue with fixed retries in the error message. I didn't find any other occurrences (so just `image scan` / `image check` / `deployment check`).